avocado/utils/ssh.py

```diff
@@ -188,11 +188,18 @@ def cmd(self, command):
 
         :param command: the command to execute over the SSH session
         :type command: str
+        :param sudo: Whether the command requires admin privileges to run, so
+                     that sudo will be prepended to the command.  The
+                     assumption here is that the user running the command has a
+                     sudo configuration such that a password won't be prompted.
+                     If that's not the case, the command will straight out
+                     fail. Default is False.
         :returns: The command result object.
         :rtype: A :class:`avocado.utils.process.CmdResult` instance.
         """
         return process.run(self.get_raw_ssh_command(command),
-                           ignore_status=True)
+                           ignore_status=True,
+                           sudo=sudo)
```

Comment:
@beraldoleal ,
Running the command on the destination with admin privileges is very useful for test writers, thanks for coming up with this idea.
However, implementation wise it seems wrong. If I turn `sudo=True` then it will run the ssh process with sudo on local host. Am I missing something?
Example, it end up calling `sudo ssh ls -la` but I want to `ssh sudo ls -la`.

Reply:
Yes, looks like you are right. My bad. I will double-check this.
Good catch.

Reply:
@wainersm and about now? What do you think?